src/vs/platform/positronActionBar/browser/components/actionBarSeparator.css

```
@@ -3,13 +3,28 @@
  *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
  *--------------------------------------------------------------------------------------------*/
 
+@keyframes positronActionBarSeparator-fadeIn {
+	0% {
+		opacity: 0;
+	}
+
+	100% {
+		opacity: 1;
+	}
+}
+
 .action-bar-separator {
 	width: 7px;
 	align-items: center;
 	justify-content: center;
 	display: flex !important;
 }
 
+.action-bar-separator.fade-in {
+	opacity: 0;
+	animation: positronActionBarSeparator-fadeIn 0.25s ease-in 0.25s 1 forwards;
+}
+
```

This can also be done without a keyframe at all. Not neccesary but just a note. 
```suggestion
.action-bar-separator {
	width: 7px;
	align-items: center;
	justify-content: center;
	display: flex !important;
}

.action-bar-separator {
  opacity: 0;
  transition: opacity 0.25s ease-in;
}

.action-bar-separator.fade-in {
  opacity: 1;
  transition-delay: 0.25s;
}
```